worlds/tunic/items.py

```diff
@@ -30,14 +30,14 @@ class TunicItemData(NamedTuple):
     "Lure x2": TunicItemData(IC.filler, 1, 11, "Consumables"),
     "Pepper x2": TunicItemData(IC.filler, 4, 12, "Consumables"),
     "Ivy x3": TunicItemData(IC.filler, 2, 13, "Consumables"),
-    "Effigy": TunicItemData(IC.useful, 12, 14, "Money", combat_ic=IC.progression),
+    "Effigy": TunicItemData(IC.useful, 12, 14, "Money", combat_ic=IC.progression | IC.deprioritized),
```

Comment:
are there any options which put effigy skip in logic? (if not, I agree with this)

Reply:
Nope!